SECURITY.md

```diff
@@ -0,0 +1,23 @@
+# Security Policy
```

Comment:
Why not use or link to https://github.com/input-output-hk/cardano-engineering-handbook/tree/main/policy/security#responsible-disclosure ?

Reply:
That is, indeed. I didn't see it as a problem but it might be painful then to maintain so I'll take a look how to improve it.